### PR TITLE
add functionality to extract custom headers

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
Headers can be extracted from both invocations as well as single action target results

### CODE REVIEW

File: pkg/updater/resultstore/resultstore.go [hunk 1]
```diff
@@ -294,6 +337,30 @@ func includeStatus(tg *configpb.TestGroup, sar *singleActionResult) bool {
...
+	var properties []*resultstorepb.Property
+	for _, p := range t.suiteProto.GetProperties() {
+		properties = append(properties, p)
+	}
+	for _, p := range t.caseProto.GetProperties() {
+		properties = append(properties, p)
+	}
+
+	for _, t := range t.suiteProto.GetTests() {
+		newTestResult := &testResult{t.GetTestSuite(), t.GetTestCase()}
```
Comment (chases2) [inline pkg/updater/resultstore/resultstore.go:358]: nit: I don't think the `&` does anything here?

**Review by chases2 — APPROVED** [review]
Looks good from a readability stance; will let someone else comment if desired.
/hold

File: pkg/updater/resultstore/resultstore.go (hunk 1 above)
Comment (sultan-duisenbay) [reply pkg/updater/resultstore/resultstore.go:358]: i think you're right - you can provide both a pointer and variable to a pointer receiver method

**Pull request merged**